manifests/repository/ubuntu.pp

```diff
@@ -0,0 +1,18 @@
+# == Class riak::repository::debian
+#
+# Install the correct package repository for Debian
+#
+class riak::repository::ubuntu {
+  ensure_packages('apt-transport-https') # packagecloud is https-only
```

Comment:
I'm not a big fan of using ensure_packages; I've had problems with it in the past. I'm not hard-vetoing it because I don't actually use Ubuntu and so it doesn't cause problems for me if it breaks, but I feel like managing how apt is configured is out of the scope of what the riak module should do.